extension/lib/shopgate/read_model/AddProductsToCart.js

```diff
@@ -1,3 +0,0 @@
-/**
```

Comment:
What is the reason for deleting this typedef?

Reply:
What about this question?

Reply:
I re-added this file.